src/_internals/itineraryLocation/ItineraryLocation.tsx

```diff
@@ -150,10 +160,30 @@ export const ItineraryLocation = ({
             className="kirk-itineraryLocation-bullet"
             type={isSmall ? BulletTypes.SMALL : BulletTypes.DEFAULT}
           />
-          {hasRoad && <div className="kirk-itineraryLocation-road" />}
+          {displayConnection && (
+            <svg width={8} height={height}>
```

Comment:
this is mostly an Icon instead of an Image. We should add it to the IconLibrary to be able to re-use it

Reply:
It's not an icon in Figma, I'd rather not add new discrepancies.